test/fixtures/configuration.js

```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-var Configuration = require('../../src/configuration.js');
+var Configuration = require('../../src/configuration.js').Configuration;
```

Comment:
was it renamed to configuration.ts?

Reply:
same below with error-message.js and others. please verify the paths are correct

Reply:
The file was renamed to `configuration.ts`, but the test code references the transpiled js version of the file.

Reply:
The same is true for `error-message.js`.

Reply:
I see. Makes sense.